src/ResourceManager/RecoveryServices/Commands.RecoveryServices.SiteRecovery.Test/AzureRm.Resources.ps1

```diff
@@ -0,0 +1,142 @@
+﻿function Get-AzureRmResourceGroup
```

Comment:
Same comment about this file as the other `AzureRm.Resources.ps1`. Please use the common version of this file and remove these duplicates.